extensions/wikia/AdEngine/webpack.config.js

```diff
@@ -0,0 +1,60 @@
+const path = require('path');
+const ExtractTextPlugin = require('extract-text-webpack-plugin');
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Replace this ES6 variable declaration with var. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-js-rules%3AConstLet)

[project-id]: wikia:app